static/app/components/guidedSteps/guidedSteps.tsx

```diff
@@ -17,7 +17,10 @@ import {space} from 'sentry/styles/space';
 import usePrevious from 'sentry/utils/usePrevious';
 
 type GuidedStepsProps = {
-  children: React.ReactElement<StepProps> | React.ReactElement<StepProps>[];
+  children:
```

Comment:
I think this can actually be React.ReactNode now - I forgot to modify the types after I changed it to stop using Children.map which required the children to be Steps